File: Beecrowd/2708 Turistas no Parque Huacachina/main.c
#include <stdio.h>
#include <string.h>

int main(){
    int t, saida = 0, volta = 0, contS = 0, contV = 0, res[2];
    char str[7];

    while(1){
        scanf("%s %d", str, &t);

        if(strcmp(str, "ABEND") == 0)
            break;
        
        if(strcmp(str, "SALIDA") == 0){
            saida += t;
            contS++;
        }
        else{
            volta += t;
            contV++;
        }

    }

    res[0] = saida - volta;
    res[1] = contS - contV;

    printf("%d\n", res[0]);
    printf("%d\n", res[1]);

    return 0;
}

File: Beecrowd/2708 Turistas no Parque Huacachina/main.py
saida, volta, contS, contV = 0, 0, 0, 0 
res = []

while(True):
    entrada = [_ for _ in input().split()] # Usar quando não tem limite min e max de entradas

    if entrada[0] == "ABEND":
        break
        
    if entrada[0] == "SALIDA":
        saida += int(entrada[1])
        contS += 1
    else:
        volta += int(entrada[1])
        contV += 1

res.append(saida - volta)
res.append(contS - contV)

print(res[0])
print(res[1])